src/DayOfWeek.js

```diff
@@ -71,12 +71,14 @@ export class DayOfWeek extends Temporal {
      * @returns {DayOfWeek}
      */
     static valueOf(name) {
-        for(var ordinal=0; ordinal < ENUMS.length; ordinal++){
+        let ordinalResult = 0;
```

Comment:
i think the additional `ordinalResult` variable is not necessary, imho it would be nicer to initialize the ordinal variable outside of the loop, modify it in the loop and then return it after the loop